modules/monitoring-about-cluster-monitoring.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring.adoc
+
+[id='monitoring-about-cluster-monitoring-{context}']
+= About cluster monitoring
+
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that is based on the link:https://prometheus.io/[Prometheus] open source project and its wider eco-system. It provides monitoring of cluster components and ships with a set of alerts to immediately notify the cluster administrator about any occurring problems and a set of link:https://grafana.com/[Grafana] dashboards.
+
+The monitoring stack includes these components:
+
+* Cluster Monitoring Operator
+* Prometheus Operator
+* Prometheus
+* Prometheus Adapter
+* Alertmanager
+* `kube-state-metrics`
+* `node-exporter`
+* Grafana
+
+The {product-title} Cluster Monitoring Operator (CMO) is the central component of the stack. It controls the deployed monitoring components and resources and ensures that they are always up to date.
+
+The Prometheus Operator (PO) creates, configures, and manages Prometheus and Alertmanager instances. It also automatically generates monitoring target configurations based on familiar Kubernetes label queries.
+
+The Prometheus Adapter exposes cluster resource metrics API for horizontal pod autoscaling. Resource metrics are CPU and memory utilization.
+
+`node-exporter` is an agent deployed on every node to collect metrics about it.
+
+The `kube-state-metrics` exporter agent converts Kubernetes objects to metrics that Prometheus can use.
+
+All the components of the monitoring stack are monitored by the stack and are automatically updated when {product-title} is updated.
+
+In addition to the components of the stack itself, the monitoring stack monitors:
+
+* `cluster-version-operator`
+* `image-registry`
+* `kube-apiserver`
+* `kube-apiserver-operator`
+* `kube-controller-manager`
+* `kube-controller-manager-operator`
+* `kube-scheduler`
+* `kubelet`
+* `monitor-sdn`
+* `openshift-apiserver`
+* `openshift-apiserver-operator`
+* `openshift-controller-manager`
+* `openshift-controller-manager-operator`
+* `openshift-svcat-controller-manager-operator`
```

Comment:
should be `openshift-service-catalog-controller-manager-operator`

Reply:
also missed following targets
`router-default`
`catalog-operator`
`olm-operator`
**********************************************************
FYI: full targets list
**********************************************************
openshift-apiserver-operator/openshift-apiserver-operator/0
openshift-controller-manager-operator/openshift-controller-manager-operator/0
openshift-controller-manager/openshift-controller-manager/0
openshift-image-registry/image-registry/0
openshift-ingress/router-default/0
openshift-kube-apiserver-operator/kube-apiserver-operator/0
openshift-kube-controller-manager-operator/kube-controller-manager-operator/0
openshift-monitoring/alertmanager/0
openshift-monitoring/cluster-monitoring-operator/0
openshift-monitoring/cluster-version-operator/0
openshift-monitoring/grafana/0
openshift-monitoring/kube-apiserver/0
openshift-monitoring/kube-controller-manager/0
openshift-monitoring/kube-scheduler/0
openshift-monitoring/kube-state-metrics/0
openshift-monitoring/kube-state-metrics/1
openshift-monitoring/kubelet/0
openshift-monitoring/kubelet/1
openshift-monitoring/kubelet/2
openshift-monitoring/node-exporter/0
openshift-monitoring/openshift-apiserver/0
openshift-monitoring/prometheus-operator/0 
openshift-monitoring/prometheus/0
openshift-monitoring/telemeter-client/0
openshift-operator-lifecycle-manager/catalog-operator/0
openshift-operator-lifecycle-manager/olm-operator/0
openshift-sdn/monitor-sdn/0
openshift-service-catalog-controller-manager-operator/openshift-service-catalog-controller-manager-operator/0
**********************************************************